website/source/docs/agent/configuration/client.html.md

```diff
@@ -90,6 +90,9 @@ client {
   receive work. This may be specified as an IP address or DNS, with or without
   the port. If the port is omitted, the default port of `4647` is used.
 
+- `server_join` <code>([ServerJoin][server_join]: nil)</code> - Specifies
+  configuration which is specific to retry joining Nomad servers.
```

Comment:
Specifies how the Nomad client will connect to Nomad servers. The `start_join` field is not supported on the client. The `retry_join` fields may directly specify the server address or use `go-discover` syntax for auto-discovery. See the documentation for more detail.

Reply:
Can you use the text I wrote